src/views/splash/presentation.jsx

```diff
@@ -31,13 +31,12 @@ const LoveProjectMessage = require('./activity-rows/love-project.jsx');
 const RemixProjectMessage = require('./activity-rows/remix-project.jsx');
 const ShareProjectMessage = require('./activity-rows/share-project.jsx');
 
-// Featured Banner Components
-const TopBanner = require('./feature/top-banner.jsx');
-const SmallTopBanner = require('./feature/small-top-banner.jsx');
-const MiddleBanner = require('./feature/middle-banner.jsx');
+// Hour of Code Banner Components
+const TopBanner = require('./hoc/top-banner.jsx');
+const MiddleBanner = require('./hoc/middle-banner.jsx');
 
-// Scratch 3.0 Launch Banner
-const LAUNCH_END_TIME = 1547873999000;
+const HOC_START_TIME = 157526280000; // 2019-12-02 00:00:00
```

Comment:
This start time is missing a 0, so that the banner will display for testing purposes. Will update it once we take a look on staging.